05.ls/ls.rb

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+
+MAX_COLUMN = 3
+SPACE_LENGTH = 3
+
+def main
+  files = search_files
```

Comment:
```suggestion
  files = search_files(ARGV[0])
  
def search_files(path)
```

みたいな設計にするとsearch_filesのコードが読みやすくなりそうです

Reply:
引数が何を表してるのか分かりやすくするためと解釈、その通りだと思うので修正しました。